CHANGELOG.md

```diff
@@ -1,12 +1,25 @@
 # librdkafka v1.6.0
 
+## Enhancements
+
+ * The generated `CONFIGURATION.md` (through `rd_kafka_conf_properties_show())`)
+   now include all properties and values, regardless if they were included in
+   the build, and setting a disabled property or value through
+   `rd_kafka_conf_set()` now returns `RD_KAFKA_CONF_INVALID` and provides
+   a more useful error string saying why the property can't be set.
+ * Consumer configs on producers and vice versa will now be logged with
+   warning messages on client instantiation.
```

Comment:
ideally, the client wouldn't even start up in this case. Maybe make this the behavior in 2.0?